### PR TITLE
Added wordcamp udaipur

### PR DESCRIPTION
**Pull request by sivaram636**
**Details of the conference**:

- Website: https://2019.udaipur.wordcamp.org/
- Date: 23 Nov
- Venue: The Artist House Near, Ashoka Cinema Rd, (Max Store) Surajpole, Udaipur
- Description: Primarily a Wordpress Conference, covering everything related to WordPress and include things such as blogging, coding, branding, and security.

### CODE REVIEW

**Comment by nikhita** [comment]
Does the conference have a CFP? I can't find it on the website :thinking:

**Comment by sivaram636** [comment]
https://2019.udaipur.wordcamp.org/2019/07/31/call-for-speakers/

File: README.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ To add a conference to the list, please [contribute](#contributing)!
 | Conference | Date | Venue | Description | Scholarship |
 |------------|------|-------|-------------|-------------|
 | [SciPy India](https://scipy.in/2019) | 29-30 Nov | Lecture Hall Complex, IIT Bombay | Annual gathering of Pythonistas providing opportunities to spread the use of the Python programming language in the Scientific Computing community in India | Yes, for groups of 5 (or more) and for IIT Bombay student and staff |
+| [WordCamp Udaipur](https://2019.udaipur.wordcamp.org/) | 23 Nov | The Artist House Near, Ashoka Cinema Rd, (Max Store) Surajpole, Udaipur | Primarily a Wordpress Conference, covering everything related to WordPress and include things such as blogging, coding, branding, and security. | No |
```
Comment (nikhita) [inline README.md]: Can you edit the venue to just `The Artist House, Surajpole, Udaipur`?

Comment (sivaram636) [reply README.md]: @nikhita I have updated.

**Pull request merged**